### PR TITLE
Add soundtrack preference and separate channel

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

- Adds a separate preference toggle for soundtrack songs. Many players do not want regular station ambiance (it can be annoying) but enjoy the soundtrack songs, myself included. Missing out on these makes me sad. Plus, toggling ambiance on doesn't play the existing soundtrack. Now it does.
- Adds an adminverb under R_SOUND that can play soundtrack songs

For merge:

DB toggle needs to be manually toggled on for existing users, due to the way preftoggles work.

```sql
UPDATE SS13_preferences SET preference_value = preference_value | (1<<13) WHERE preference_tag = 2;
```

## Why It's Good For The Game

The soundtrack is super fun, and having more people get the opportunity to hear it is great.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/10366817/223946057-3904e8df-ec6a-4876-9a05-52a3d7a810b0.mp4

</details>

## Changelog
:cl:
add: Added a preference toggle for soundtrack music (Countdown, Mind Crawler, etc)
admin: Added an admin verb for playing soundtrack music.
tweak: Un-silencing soundtrack music will now start playing the music again rather than missing out entirely.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
I like the song but I don't really feel like it fits the theme enough

**Comment by Chirynnn** [comment]
Hell yeah it does, also I imagine the soundtrack is separate from the ambience sound, so if you suddenly go into space it won't stop playing no?

**Comment by Crossedfall** [comment]
The song is fine, it's just a bad fit. That's why it's a bonus track. We had a pretty lengthy discussion over this already. 🔑

**Comment by itsmeow** [comment]
If someone else keys this don't close, I'll just remove the track

**Comment by itsmeow** [comment]
Removed 'Malf AI'

**Comment by Crossedfall** [comment]
Add the required SQL update to the DB changelog, please. No need to bump the schema versioning.

File: SQL/database_changelog.txt [hunk 1]
```diff
@@ -8,6 +8,14 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 1);
 
 In any query remember to add a prefix to the table names if you use one.
 
+-----------------------------------------------------
+
+Version 6.1-SOUNDTRACK - 11 March 2023 by itsmeow - Minor update, enable soundtrack pref for existing users
+
+UPDATE SS13_preferences SET preference_value = preference_value | (1<<13) WHERE preference_tag = 2;
+
+-----------------------------------------------------
```
Comment (PowerfulBacon) [inline SQL/database_changelog.txt:17]: Can't we just use the version based prefs updating, or was that removed?

Comment (itsmeow) [reply SQL/database_changelog.txt:17]: That was for savefiles

**Review by Crossedfall — APPROVED** [review]
As far as the prefs portion of this is concerned, it all looks fine.

**Review by PowerfulBacon — APPROVED** [review]

File: code/game/sound.dm [hunk 2]
```diff
@@ -190,18 +190,29 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 			var/mob/M = m
 			M.playsound_local(M, null, volume, vary, frequency, null, channel, pressure_affected, S)
 
-/proc/play_soundtrack_music(var/datum/soundtrack_song/song, list/hearers = null, volume = 80, ignore_prefs = FALSE, play_to_lobby = FALSE, allow_deaf = TRUE, only_station = FALSE)
-	var/sound/S = sound(initial(song.file), volume=volume, wait=0, channel=CHANNEL_AMBIENT_MUSIC)
+// only_station:
+// 0: respect soundtrack option
+// 1: play to all
+// 2: play only on station
```
Comment (Archanial) [inline code/game/sound.dm]: define those

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/game/sound.dm (hunk 2 above)
Comment (itsmeow) [reply code/game/sound.dm]: uugh it's such a waste

Comment (itsmeow) [reply code/game/sound.dm]: it's only usable for an admin proc call where defines arent usable anyway

Comment (Archanial) [reply code/game/sound.dm]: thank you ‼️

**Review by Archanial — APPROVED** [review]

**Comment by Crossedfall** [comment]
DB changes will be applied shortly in post.

**Pull request merged**